Move vpc example to example directory (#1)
klaytn/klaytn-terraform: serivce-chain-aws/README.md

A terraform module to create a Klaytn service chain network including SCN nodes and main chain EN nodes. It help you to build infrustcture on AWS. After create instance nodes with this module, you may can install KSCN, KEN with [klaytn-ansible](https://github.com/klaytn/klaytn-ansible) project under Klaytn organization. For more details regarding Klaytn service chains, please refer to [Klaytn Docs](https://docs.klaytn.com/node/service-chain).

## Usage

> For example, if your configuration looks like the following:

Reviewer: How about this? (please let me know I misunderstood)

```suggestion
You may use the module with the following configuration (for example):
```

}
```

> Or simply edit [terraform.tfvars](https://github.com/klaytn/klaytn-terraform/blob/master/serivce-chain-aws/terraform.tfvars) and fill each elements. Then run `terraform init / plan / apply` to create resources in AWS.

Reviewer: ```suggestion
Or simply edit [terraform.tfvars](https://github.com/klaytn/klaytn-terraform/blob/master/serivce-chain-aws/terraform.tfvars) and fill each element. Then run `terraform init / plan / apply` to create resources in AWS.
```

## with VPC module example

> This module not create network layers such as VPC/Subnet. It could be done with [terraform-aws-vpc module](https://github.com/terraform-aws-modules/terraform-aws-vpc.git) which provide on terraform-aws-modules organization. A example is contained in the [examples/with_vpc](https://github.com/klaytn/klaytn-terraform/tree/master/serivce-chain-aws/examples/with_vpc) directory.

Reviewer: ```suggestion
This module does not create network resourcse such as the VPC/subnet. This could be done with [terraform-aws-vpc module](https://github.com/terraform-aws-modules/terraform-aws-vpc.git) which is provided in the terraform-aws-modules organization. A example is contained in the [examples/with_vpc](https://github.com/klaytn/klaytn-terraform/tree/master/serivce-chain-aws/examples/with_vpc) directory.
```
